### PR TITLE
fix(atlantis): remove additional spaces in pod annotation

### PR DESCRIPTION
**Pull request by anasinnyk**
Signed-off-by: Andrii Nasinnyk <a.nasinnyk@globalgames.net>

#### What this PR does / why we need it:
Remove additional spaces. Pod annotations don't work.
Rename cmd to command in container describe, because cmd not used. (https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#container-v1-core)
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/atlantis/templates/statefulset.yaml [hunk 1]
```diff
@@ -22,10 +22,10 @@ spec:
       labels:
         app: {{ template "atlantis.name" . }}
         release: {{ .Release.Name }}
-  {{- with .Values.podTemplate.annotations }}
+      {{- with .Values.podTemplate.annotations }}
       annotations:
-  {{ toYaml . | indent 10 }}
-  {{- end }}
+{{ toYaml . | indent 10 }}
```
Comment (davidkarlsen) [inline stable/atlantis/templates/statefulset.yaml]: Should this in fact indent 8?

Comment (sstarcher) [reply stable/atlantis/templates/statefulset.yaml]: ^ verified 8 is correct

**Review by sstarcher — CHANGES_REQUESTED** [review]
thanks for fixing the mistakes

File: stable/atlantis/templates/statefulset.yaml (hunk 1 above)
Comment (anasinnyk) [reply stable/atlantis/templates/statefulset.yaml]: done

**Review by sstarcher — APPROVED** [review]

**Comment by anasinnyk** [comment]
@k8s-ci-robot doesn't hear you : (

**Pull request merged**